tests/unit/test_sentenceHelper.py

```diff
@@ -0,0 +1,39 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Rob Meredith
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import unittest
+import documentNavigation.sentenceHelper as sentenceHelper
+
+
+class TestSentenceHelper(unittest.TestCase):
+	def test_findEndOfSentence(self):
+		westernSentences = (
+			"Test sentence number one. "
+			"Test sentence number two?       "
+			"Test sentence number three!"
+		)
+		fullWidthSentences = (
+			"Test sentence number one\u3002"
+			"Test sentence number two\uff01"
+			"Test sentence number three\uff1f"
+		)
+		notASentence = "not a terminated sentence"
+		emptyString = ""
+		res = sentenceHelper._findEndOfSentence(westernSentences, 0)
+		self.assertEqual(res, 26)
```

Comment:
Is there a way to make this easier to verify? Reading this I find myself asking, "what is the character at index 26, and how do I know if that is right?"

How about:
``` python
first = "Test sentence number one."
...
westernSentences = (
	# add whitespace to test XYZ, what boundary conditions does this test?
	f"{first} {second}        {third}"
)
...
self.assertEqual(
	res,
	len(first) + 1  # expected index: one past the end of the sentence (after punctuation.)
)
```

Reply:
@feerrenrut Yes, paragraph navigation tests to be developped next week. I thought I would start with the simplest module first.

Reply:
@feerrenrut Any thoughts on my unit tests?